Geodatabase Data Curation Primer/Geodata-Primer.md

```diff
@@ -250,7 +250,7 @@ The two most common standards for geospatial metadata (which encompass geodataba
 
 The second is the Content Standard for Digital Geospatial Metadata (CSDGM) developed by the United States Government’s Federal Geographic Data Committee (FGDC). United States federal government datasets typically use this standard, though the FGDC does endorse the use of ISO 191xx standards. This standard is being used less and less outside of the federal government as the ISO standards become more prevalent.
 
-While the ISO 191xx and FGDC CSDGM standards differ in organization and element requirements, the two content standards are fairly similar. In addition to core metadata elements typically required or recommended by repositories (e.g. title, creator, date, subject, description, rights, etc.), both standards require the following elements for geospatial materials:
```

Comment:
Technically ISO 1911x only requires 7 elements (title, abstract, dataset reference date, topic category, metadata contact, metadata date, and dataset language)